main.go

```diff
@@ -44,22 +43,57 @@ const (
+	MinNakedHex     int  `toml:"min_naked_hex"`  // ignore words at least this long if only hex digits.
+	MakeSuggestions int  `toml:"suggest"`        // make suggestions for misspelled words.
+}
+
+var defaults = config{
+	Show:            true,
+	CheckStrings:    false,
+	IgnoreUpper:     true,
+	ignoreSingle:    true,
+	IgnoreNumbers:   true,
+	MaskURLs:        true,
+	CamelSplit:      true,
+	MaxWordLen:      40,
+	MinNakedHex:     8,
+	MakeSuggestions: never,
+}
+
 func gospel() (status int) {
-	show := flag.Bool("show", true, "print comment or string with misspellings")
-	checkStrings := flag.Bool("check-strings", false, "check string literals")
-	ignoreUpper := flag.Bool("ignore-upper", true, "ignore all-uppercase words")
-	ignoreSingle := flag.Bool("ignore-single", true, "ignore single letter words")
+	config, status, err := loadConfig()
+	if err != nil {
+		fmt.Fprintln(os.Stderr, err)
+		return status
+	}
+
+	flag.BoolVar(&config.Show, "show", config.Show, "print comment or string with misspellings")
+	flag.BoolVar(&config.CheckStrings, "check-strings", config.CheckStrings, "check string literals")
+	flag.BoolVar(&config.IgnoreUpper, "ignore-upper", config.IgnoreUpper, "ignore all-uppercase words")
+	flag.BoolVar(&config.ignoreSingle, "ignore-single", config.ignoreSingle, "ignore single letter words")
+	flag.BoolVar(&config.IgnoreNumbers, "ignore-numbers", config.IgnoreNumbers, "ignore Go syntax number literals")
+	flag.BoolVar(&config.MaskURLs, "mask-urls", config.MaskURLs, "mask URLs in text")
+	flag.BoolVar(&config.CamelSplit, "camel", config.CamelSplit, "split words on camel case")
+	flag.IntVar(&config.MinNakedHex, "min-naked-hex", config.MinNakedHex, "length to recognize hex-digit words as number (0 is never ignore)")
+	flag.IntVar(&config.MaxWordLen, "max-word-len", config.MaxWordLen, "ignore words longer than this (0 is no limit)")
+	flag.IntVar(&config.MakeSuggestions, "suggest", config.MakeSuggestions, "make suggestions for misspellings (0 - never, 1 - first instance, 2 - always)")
 	ignoreIdents := flag.Bool("ignore-idents", true, "ignore words matching identifiers")
-	ignoreNumbers := flag.Bool("ignore-numbers", true, "ignore Go syntax number literals")
-	maskURLs := flag.Bool("mask-urls", true, "mask URLs in text")
-	camelSplit := flag.Bool("camel", true, "split words on camel case")
-	minNakedHex := flag.Int("min-naked-hex", 8, "length to recognize hex-digit words as number (0 is never ignore)")
-	maxWordLen := flag.Int("max-word-len", 40, "ignore words longer than this (0 is no limit)")
-	suggest := flag.Int("suggest", 0, "make suggestions for misspellings (0 - never, 1 - first instance, 2 - always)")
-	words := flag.String("misspellings", "", "file to write a dictionary of misspellings (.dic format)")
-	update := flag.Bool("update-dict", false, "update misspellings dictionary instead of creating a new one")
 	lang := flag.String("lang", "en_US", "language to use")
```

Comment:
It will be. I want to factor out the dictionary handling into a type that's distinct from the checker. The dictionary type will use the remaining 3ish config options that make sense to persist (`lang`, `ignore-idents` and `dict-paths`).